docs/scos/dev/glue-api-guides/202311.0/conventions/default-conventions.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Default conventions
+description: This document will describe what convention Spryker provides OOTB
+last_updated: December 12, 2023
+template: howto-guide-template
+---
+
+Spryker provides some of the [conventions](/docs/scos/dev/glue-api-guides/{{page.version}}/conventions/what-is-api-convention.html) OOTB
+
+## Available conventions
+
+### JSON:API
+
+JSON:API is available for usage OOTB in Spryker. In order to use it you need to implement the `Spryker\Glue\GlueJsonApiConventionExtension\Dependency\Plugin\JsonApiResourceInterface` interface in your resource plugin.
```

Comment:
Why do I need to implement something when it is OOTB? probably you want to say that JSON is default API format in glue communication but if I need to implement something first before I can use it, is not my understanding of OOTB

Reply:
Provided an example